setup.py

```diff
@@ -187,16 +179,9 @@ def generate_file_link(file_name):
         if not os.path.exists(dst):
             os.symlink(src, dst)
 
-    @staticmethod
-    def create_paths():
-        """Method used to create the paths used by Kytos in develop mode."""
-        directories = [os.path.join(BASE_ENV, 'etc/kytos')]
-        for directory in directories:
-            if not os.path.exists(directory):
-                os.makedirs(directory)
-
 
-requirements = [i.strip() for i in open("requirements.txt").readlines()]
+setup_requires = ['jinja2']
```

Comment:
I'm going to accept this, because I need to see the things moving on. But this must be fixed.